packages/mobile/src/navigator/Navigator.tsx

```diff
@@ -527,18 +499,81 @@ const modalScreenOptions: ScreenOptions = Platform.select({
   }),
 })
 
+const animatedScreens = (Navigator: typeof Stack) => (
```

Comment:
nit: how about we rename them to `modalAnimatedScreens`? And maybe move `SelectCountry` to it too.